client/app.vue

```diff
@@ -36,3 +36,69 @@ useEventBus(LoggedOutEvent).on(() => {
     </NuxtLayout>
   </div>
 </template>
+
+<style lang="scss">
+.layout-in-enter-active,
+.layout-in-leave-active {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  transition: all 0.4s cubic-bezier(0.87, 0, 0.13, 1);
+
+  .p-card {
+    transition: all 0.4s cubic-bezier(0.87, 0, 0.13, 1);
+  }
```

Comment:
create a variable for the curve (or the whole transition), since they will both always be the same